mysql-test/suite/innodb/t/doublewrite_debug.test

```diff
@@ -44,7 +44,10 @@ commit work;
 # Slow shutdown and restart to make sure ibuf merge is finished
 SET GLOBAL innodb_fast_shutdown = 0;
 let $shutdown_timeout=;
+let $restart_parameters="--debug_dbug=+d,ib_log_checkpoint_avoid_hard";
 --source include/restart_mysqld.inc
+--source ../include/no_checkpoint_start.inc
+set global innodb_flush_sync=0;
```

Comment:
I think that we should swap these. The "critical section" for checkpoint is the transaction that follows these lines.